tools/diffdeb

```diff
@@ -37,16 +53,13 @@ function unpack_and_remove_version() {
     rm deb
 
     # Unzip data.tar to remove the gzip timestamp from the header
-    gunzip data.tar.gz
+    $TAR xfz data.tar.gz
```

Comment:
Please update the comments on line 46–49 and line 55.

Reply:
committed to dev as 12a180a7 (oops)